realsense2_camera/src/realsense_node_factory.cpp

```diff
@@ -128,9 +126,82 @@ void RealSenseNodeFactory::onInit()
                     {
                     if (info.was_removed(_device))
                     {
-                    ROS_FATAL("The device has been disconnected! Terminating RealSense Node...");
-                    ros::shutdown();
-                    exit(1);
+                    ROS_WARN("The device has been disconnected! Looking for a new RealSense...");
```

Comment:
> Looking for a new RealSense...

I think I see what this is trying to say, but this could give an impression that the Realsense device would have to be different one than the one just got disconnected.